tools/devctr/Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV LC_ALL=C.UTF-8
 RUN apt-get update \
     && apt-get -y install --no-install-recommends \
         # essential build tools
-        gcc make libc-dev binutils-dev libssl-dev \
+        gcc make libc-dev binutils-dev libssl-dev gdbserver \
```

Comment:
nit: I would put it in the `        # needed for integration tests` section, although that won't be quite true

Reply:
I put it in a "useful utilities" section for now, if it doesnt fix any of the existing ones :sweat_smile: